python/humbug/report.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         bugout_token: Optional[str] = None,
         bugout_journal_id: Optional[str] = None,
         timeout_seconds: int = 10,
+        tags: Optional[List[str]] = None
```

Comment:
Can you also add the following two parameters:
1. `client_id: str`,
2. `session_id: str`.

Reply:
Was added in v0.1.3